main/management/commands/start.py

```diff
@@ -31,9 +32,22 @@ def handle(self, *args, **options):
             logging.error('Database could not be found, exiting.')
             sys.exit(1)
 
-        print('Running migrations')
-        call_command("migrate")
-        print('Running setup')
+        attempts_left = 5
+        while attempts_left:
+            try:
+                logging.info('Trying to run migrations')
+                call_command("migrate")
+                logging.info('Migrations ran')
+                break
+            except OperationalError:
+                attempts_left -= 1
+                logging.warning('Cannot run migrations, retrying.')
+                time.sleep(0.5)
```

Comment:
I think this exception is too wide. Many problems can lead to OperationalErrors.

I think you should be catching specifically the OperationalError where the `database is starting up` message is shown. For any other type of OperationalError, I think you should still fail. It's important to narrow the scope for exceptions as small as you possibly can, so you don't accidentally catch errors that are unexpected, otherwise those errors may fail silently and cause you a massive headache.

I also think the warning message is too general. Once this only catches the database is starting error, the warning message could say "Database not ready yet, retrying"